core/nodeFilesystemShared/src/files/FileSystemNodeJs.kt

```diff
@@ -124,6 +124,7 @@ public actual val SystemTemporaryDirectory: Path
         return Path(os.tmpdir() ?: "")
     }
 
+@Suppress("EXPECT_ACTUAL_CLASSIFIERS_ARE_IN_BETA_WARNING")
```

Comment:
should be not needed after https://github.com/Kotlin/kotlinx-io/pull/325 was merged